components/otp-input/src/ReuseOTPInput.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import { HeadlessOTPInput } from "@locoworks/reusejs-react-otp-input";
+import { HeadlessOTPInputProps } from "./HeadlessOTPInput";
+
+export interface ReuseOTPInputProps extends HeadlessOTPInputProps {}
+
+const ReuseOTPInput: React.FC<ReuseOTPInputProps> = ({
+  value = "",
+  numInputs = 4,
+  onChange,
+  renderInput,
+  shouldAutoFocus = false,
+  inputType = "text",
+  renderSeparator,
+  placeholder,
+  containerStyle,
+  inputStyle,
+}) => {
+  const [otp, setOtp] = useState("");
+
+  return (
+    <div className="flex flex-col items-center gap-x-3 justify-center py-10 mt-10 border rounded bg-gray-50">
+      <HeadlessOTPInput
+        value={otp}
+        onChange={setOtp}
+        numInputs={5}
+        renderSeparator={null}
+        renderInput={(props: any) => <input {...props} />}
+        inputStyle={{
```

Comment:
How do we change these styles while implementing?

Reply:
@ShreyDhyani  Which component we will use in applications(Wiredup)? Is it a Reuse component or Headless Component?

I thought we will use the Headless component directly in applications like Wiredup.

Reply:
@Subbu-betalectic Adding any component to ReuseJS-React is not dependent on which version you use or which project you use it in. Following criteria must be met for adding a component here:

1. Atleast 2 versions should be exposed, Headless, Reuse.
2. Headless Version: No default styling, generalised props, user should be able to override components where ever possible.
3. Reuse Version: Default styling (preferrebly tailwind classes), which can be overridden by props.

 
Use twMerge from tailwind-merge to resolve final classes. 
Refer here for how to use twMerge and create default classes: https://github.com/locoworks/reusejs-react/blob/main/components/input-group/src/ReuseInputGroup.tsx